org.jacoco.doc/docroot/doc/changes.html

```diff
@@ -22,6 +22,9 @@ <h2>Snapshot Build @qualified.bundle.version@ (@build.date@)</h2>
 
 <h3>New Features</h3>
 <ul>
+  <li>Do not filter out synthetic methods generated by Kotlin compiler that
+      contain values of default arguments
+      (GitHub <a href="https://github.com/jacoco/jacoco/issues/774">#774</a>).</li>
```

Comment:
Maybe this should be `Fixed Bugs` @marchof WDYT?

Reply:
Strictly speaking this is a bug. Even if nobody noticed it ;)

Maybe it is more readable in a positive way: "Report code coverage correctly for Kotlin methods with default arguments"

Reply:
Updated.

But maybe this description forces us to really fix/workaround  line number weirdness.